Docs: Added extended Home Assistant example in documentation

Wetzel402 opened the pull request:
<!--
  This template provides some ideas of things to include in your PR description.

  To start, try providing a short summary of your changes in the Title above. We follow Conventional Commits syntax, please ensure your title is prefixed with one of:

  - `docs: ` Added extended Home Assistant example

  If a section of the PR template does not apply to this PR, then delete that section.

  PLEASE READ:
  -------------------------
  Mealie is moving to a regular, automatic release schedule. This means that all PRs should be in a
  stable state, ready for release. This includes:

  - Ensuring new tests have been added to cover new features, or to prevent regressions.
  - Work is fully complete and usable

 -->

## What type of PR is this?

_(REQUIRED)_

<!--
  Delete any of the following that do not apply:
 -->

- documentation
Added extended Home Assistant example

Wetzel402 committed:
Update home-assistant.md

Wetzel402 committed:
Add files via upload

Wetzel402 committed:
Merge branch 'mealie-next' into mealie-next

Wetzel402 committed:
Update home-assistant.md

michael-genson commented on docs/docs/documentation/community-guide/home-assistant.md:
````diff
@@ -83,3 +83,102 @@ card_mod:
+
+```yaml
+rest:
+  - resource_template: http://mealie:9000/api/groups/mealplans?start_date={{ now().strftime('%Y-%m-%d') }}&page=1&perPage=5&orderBy=date&orderDirection=asc
````
```suggestion
  - resource_template: http://mealie:9000/api/households/mealplans?start_date={{ now().strftime('%Y-%m-%d') }}&page=1&perPage=5&orderBy=date&orderDirection=asc
```

michael-genson reviewed (changes requested):
I can't suggest changes on line 27 and 39, but could you also update those from `.../groups/mealplans...` to `.../households/mealplans...`? The new households PR changes those endpoints

joostlek commented on docs/docs/documentation/community-guide/home-assistant.md:
````diff
@@ -83,3 +83,102 @@ card_mod:
+
+Make sure the url and port (`http://mealie:9000` ) matches your installation's address and _API_ port. Configure the page (`1`), perPage (`5`), and orderDirection (`asc`) as desired. This example will return 1 page with 5 meals per page in ascending order starting today to exclude yesterday's meals. 
+
+```yaml
````
Should we change this to using the integration?

Wetzel402 replied:
I think the wording from the previous users is accurate and I'm just reusing it.  We are creating REST sensors for HA.  In HA integration typically means something more advanced though `sensor.rest` is technically an integration too. 

https://www.home-assistant.io/integrations/sensor.rest/
https://www.home-assistant.io/integrations

joostlek replied:
Yes but I'm talking about using the mealie integration to communicate with Mealie.

Wetzel402 replied:
Is there a HACS integration?

michael-genson replied:
Mealie is an official integration now: https://www.home-assistant.io/integrations/mealie/

Wetzel402 replied:
Considering there is an official integration I may have to rewrite the documentation for this PR.  I will have to explore the integration.  As it stands now my PR doesn't use the integration at all but rather the API and REST sensors.

joostlek replied:
I did add services that you can use to request stuff and then use a template trigger to be able to create template entities from that response. It's a bit more complicated than having ready to go entities, but it is very flexible now

Wetzel402 committed:
modified:   frontend/components/global/BaseButtonGroup.vue
    modified:   frontend/pages/group/data/labels.vue
	modified:   frontend/pages/shopping-lists/_id.vue
	modified:   mealie/schema/labels/multi_purpose_label.py

Wetzel402 committed:
Merge branch 'mealie-recipes:mealie-next' into mealie-next

Wetzel402 committed:
Update docs/docs/documentation/community-guide/home-assistant.md

Co-authored-by: Michael Genson <71845777+michael-genson@users.noreply.github.com>
